### PR TITLE
Adding Example section in README

### PR DESCRIPTION
**Pull request by Kuljeet-123**
Fixes issue:- #60 .

### CODE REVIEW

**Comment by Parikshit-Hooda** [comment]
@coderwassananmol @rkpattnaik780 currently the video are in the repo itself. I will upload them to youtube eventually. Please approve this for now so that we may move fwd.

File: README.md [hunk 1]
```diff
@@ -61,6 +61,25 @@ docker-compose start
 - Run `npm run dev` for development and `npm run start` for production.
 - Open your browser and navigate to http://localhost:5000
 
+### Example
+  This is the simple Example how BUB2(Book Uploader Bot) works.
```
Comment (rkpattnaik780) [inline README.md]: I can see incorrect capitalization throughout the section. Please fix them. You can use `` for highlighting purposes if incase you have capitalized for highlighting.

Comment (Kuljeet-123) [reply README.md]: @rkpattnaik780  please review it and tell me if it is right or wrong.

**Comment by Kuljeet-123** [comment]
@rkpattnaik780  I have updated the PR please review it.

**Comment by Kuljeet-123** [comment]
@rkpattnaik780 I have updated the PR please review it and tell me if it is right or wrong.

**Pull request merged**